style.css

```diff
@@ -410,3 +413,117 @@ button:hover {
   background: #6070ff;
   box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
 }
+
+/* Form */
+#contact-me {
+  border: 1px solid transparent;
+  background: #6070ff;
+  border-radius: 100px 0 0 0;
+  background-image: url(Assets/contact-mee.png);
+  background-position: right top;
+  background-repeat: no-repeat;
+  margin-top: 145px;
+  padding-top: 90px;
```

Comment:
A great change in the implementation of the previous reviewer. But still, you should make changes to make it the same as the Figma one.
Your design
![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/101523582/795e59fa-eb57-4c22-bec4-2bf0f15c3646)

Figma design
![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/101523582/fa73cacc-1186-432b-8a0a-010373331e2f)

Kindly make changes to the border-radius and the right top corner of your app.